### ISSUE TITLE
Add option to disable new Column Mode behavior and revert to previous

### ISSUE THREAD

**Issue by brandishwar** [notepad-plus-plus/notepad-plus-plus#14521]
With v8.6, the "column mode" behavior was changed such that column mode is no longer automatically exited merely by pressing an arrow key. I very much preferred that old behavior since I didn't need to press Esc (or any additional key) to get out of column mode - something pointed out in #14492 doesn't always work right - before I could move the cursor again to start a new selection - whether in Column Mode or not.

As such, I request an option be available in Settings->Preferences so I can turn off the new behavior.

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/aef04381800c920264dc475483f5013572dfea5a

**Comment by brandishwar** [comment]
That PR does NOT address this request. Your PR, @donho, requires creating a blank file to disable the new functionality. And I'm requesting an option be added to Settings->Preferences, or someplace else in the user interface.

**Comment by donho** [comment]
>  I'm requesting an option be added to Settings->Preferences, or someplace else in the user interface.

The answer is https://github.com/notepad-plus-plus/notepad-plus-plus/commit/aef04381800c920264dc475483f5013572dfea5a, you take it or not.

**Comment by brandishwar** [comment]
No that isn't the answer. Since, again, that is not what I'm requesting. Again I'm requesting that be added to the user interface. So, no, I won't be taking that, nor will I be accepting your "take it or not" response.

**Comment by alankilborn** [comment]
>I'm requesting that be added to the user interface. 

And your request was denied.

>I won't be taking that, nor will I be accepting your "take it or not" response.

I'm really not sure what power you think you have.
The project owner has declined your request.
You're done and out of options.

**Comment by brandishwar** [comment]
> And your request was denied.

It was closed linking to a commit, which gave me the impression it had already been implemented. Except it wasn't. That's why I requested this be re-opened. That way someone else could implement it - or I could try to take it on.

> I'm really not sure what power you think you have.

Look I was just making a request, and I'm looking for it to be reopened since the commit he linked isn't what I requested. Since, again, that leaves the request open for someone else - or me - to implement. And if he isn't going to accept anyone trying to implement this and submit a PR for it, then he can clearly say so.
